partner_phonecall_schedule/views/res_partner_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">Partner phonecall schedule</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook">
+                <page string="Phone calls" name="phonecalls">
```

Comment:
Do you think this deserves a new tab? I would put it in a many2many_tags widget inside "Sales & Purchases".

Reply:
I agree that the schedules would fit best in a widget. Anyway, the aggregated phonecall schedule needs plenty of space to show all the available ones:
![image](https://user-images.githubusercontent.com/5040182/30490766-3df2fff2-9a3b-11e7-9a1f-3628b042640d.png)

Reply:
You nailed it. Besides there's a UI problem: to manage available `resource.calendar` records, you need to have admin permissions, and `many2many_tabs` wouldn't allow the user to click & edit a calendar record as the list widget does.

So with this widget, they can edit it if anything went wrong, and still there's no need to move the resource menus or give extra privileges to users.

Reply:
OK, so if this is required, drop a comment in the XML file for explaining it to avoid the same future comments